src/app/park-form/park-form.component.ts

```diff
@@ -89,13 +90,21 @@ export class ParkFormComponent implements OnInit, OnDestroy {
   }
 
   async submitForm() {
-    const message = `<strong>Name:</strong></br>` + this.parkForm.get('name').value +
-      `</br></br><strong>Status:</strong></br>` + this.getParkInfoString('status') +
-      `</br></br><strong>Visibility:</strong></br>` + this.getParkInfoString('visible') +
-      `</br></br><strong>Capacity:</strong></br>` + this.parkForm.get('capacity').value +
-      `</br></br><strong>Link to BC Parks Site:</strong></br>` + this.parkForm.get('bcParksLink').value +
-      `</br></br><strong>Description:</strong></br>` + this.parkForm.get('description').value +
-      `</br></br><strong>Link to map:</strong></br>` + this.parkForm.get('mapLink').value;
+    const message =
```

Comment:
Are we going to include ORCS in this alert modal?

Reply:
Not yet - future change.